### PR TITLE
Fixes #1200 - Spelling and Grammatical Errors

### LINKED ISSUE #1200: 'identity or token do not match' --> 'identity or token does not match'
I see many instances of this error today: **"identity or token do not match"**

This issue is not concerned with why the error is happening. I'll write that up separately.  This issue concerns a couple grammar and spelling errors that are revealed by the error that may as well be fixed. 

I'll copy an excerpt from the log issued by ./peer peer as started from a Vagrant session.  Note two errors in the log excerpt:

1) retrieveing -> retrieving (may as well fix this spelling error while you are in the code)
2)  identity or token do not -> identify or token does not   (grammar error)

"

> 15:09:52.081 [crypto] registerCryptoEngine -> ERRO 020 [client.lukas] Failed **retrieveing** enrollment data [rpc error: code = 2 desc = "**identity or token do not match**"].
> 15:09:52.082 [crypto] register -> ERRO 021 [client.lukas] Failed registering node crypto engine [rpc error: code = 2 desc = "identity or token do not match"].
> 15:09:52.082 [crypto] Login -> ERRO 022 Failed registering client [lukas] with name [lukas] [rpc error: code = 2 desc = "**identity or token do not match**"].
> 15:09:52.083 [rest] Register -> ERRO 023 Error on client login: rpc error: code = 2 desc = '**identity or token do not match**'
> 15:09:52.112 [rest] Register -> INFO 024 REST client login...
> 15:09:52.114 [rest] Register -> INFO 025 Local data store for client loginToken: /var/hyperledger/production/client/
> 15:09:52.114 [rest] Register -> INFO 026 Logging in user 'diego' on REST interface...
> 15:09:52.114 [crypto] Login -> INFO 027 Registering client [diego] with name [diego]...
> 15:09:52.144 [crypto] getEnrollmentCertificateFromECA -> ERRO 028 [client.diego] Failed invoking CreateCertficatePair [rpc error: code = 2 desc = "**identity or token do not match**"].15:09:52.081 [crypto] registerCryptoEngine -> ERRO 020 [client.lukas] Failed **retrieveing** enrollment data [rpc error: code = 2 desc = "identity or token do not match"].
> 15:09:52.082 [crypto] register -> ERRO 021 [client.lukas] Failed registering node crypto engine [rpc error: code = 2 desc = "**identity or token do not match**"].
> 15:09:52.082 [crypto] Login -> ERRO 022 Failed registering client [lukas] with name [lukas] [rpc error: code = 2 desc = "identity or token do not match"].
> 15:09:52.083 [rest] Register -> ERRO 023 Error on client login: rpc error: code = 2 desc = 'identity or token do not match'
> 15:09:52.112 [rest] Register -> INFO 024 REST client login...
> 15:09:52.114 [rest] Register -> INFO 025 Local data store for client loginToken: /var/hyperledger/production/client/
> 15:09:52.114 [rest] Register -> INFO 026 Logging in user 'diego' on REST interface...
> 15:09:52.114 [crypto] Login -> INFO 027 Registering client [diego] with name [diego]...
> 15:09:52.144 [crypto] getEnrollmentCertificateFromECA -> ERRO 028 [client.diego] Failed invoking CreateCertficatePair [rpc error: code = 2 desc = "identity or token do not match"].

### PR DESCRIPTION
**Pull request by jeromefroe**
Hi! Just wanted to start off by saying I think this an awesome project! I was looking to get involved and I as I familiarize myself with the codebase I thought I could lend a hand by making the few small spelling and grammatical fixes requested by issue #1200. Thanks!

Think I need this for the legal obligations:

Signed-off-by: Jerome Froelich jeromefroelich@hotmail.com

### CODE REVIEW

**Comment by jeromefroe** [comment]
My apologies, I just noticed after I pushed my changes that `goimports` had mistakenly removed the "go-logging" package (I think because the package name "go-logging" is different from the name used in the code "logging"). So the second commit adds the package back to asset_management.go and asset_management_with_roles.go. Sorry for the mixup. Thanks again!

File: examples/chaincode/go/asset_management/asset_management.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ package main
 import (
 	"errors"
 	"fmt"
-
+
```
Comment (ghaskins) [inline examples/chaincode/go/asset_management/asset_management.go]: Whitespace damage?  I think if you drop this hunk, the rest looks good

**Comment by jeromefroe** [comment]
@ghaskins I just pushed a new commit trying to fix the whitespace discrepancy, please let me know if that is what you meant. In all honesty, I was a little uncertain how to fix it.

**Comment by ghaskins** [comment]
@jeromefroe it looks "worse" but its a nitpick anyway.  Don't worry about it.  Thank you for the corrections!

LGTM

Reviewed-by: Greg Haskins gregory.haskins@gmail.com

**Comment by sykesm** [comment]
> @sykesm I'll document where necessary then.

I may be wrong but I think it’s less documenting yaml than fixing all of the bad examples. The documentation has unquoted scalar pins all over the place.

**Pull request merged**